featuretools/tests/primitive_tests/test_rolling_primitive_utils.py

```diff
@@ -442,31 +542,41 @@ def count_wrapper(sub_s):
 def test_roll_series_with_gap_incorrect_types(rolling_series_pd):
     error = "Window length must be either an offset string or an integer."
     with pytest.raises(TypeError, match=error):
-        roll_series_with_gap(rolling_series_pd, window_size=4.2, gap=4)
+        roll_series_with_gap(rolling_series_pd, window_size=4.2, gap=4, min_periods=1),
 
     error = "Gap must be either an offset string or an integer."
     with pytest.raises(TypeError, match=error):
-        roll_series_with_gap(rolling_series_pd, window_size=4, gap=4.2)
+        roll_series_with_gap(rolling_series_pd, window_size=4, gap=4.2, min_periods=1)
 
 
 def test_roll_series_with_gap_negative_inputs(rolling_series_pd):
     error = "Window length must be greater than zero."
     with pytest.raises(ValueError, match=error):
-        roll_series_with_gap(rolling_series_pd, window_size=-4, gap=4)
+        roll_series_with_gap(rolling_series_pd, window_size=-4, gap=4, min_periods=1)
 
     error = "Gap must be greater than or equal to zero."
     with pytest.raises(ValueError, match=error):
-        roll_series_with_gap(rolling_series_pd, window_size=4, gap=-4)
+        roll_series_with_gap(rolling_series_pd, window_size=4, gap=-4, min_periods=1)
 
 
 def test_roll_series_with_non_offset_string_inputs(rolling_series_pd):
     error = "Cannot roll series. The specified gap, test, is not a valid offset alias."
     with pytest.raises(ValueError, match=error):
-        roll_series_with_gap(rolling_series_pd, window_size="4D", gap="test")
+        roll_series_with_gap(
+            rolling_series_pd,
+            window_size="4D",
+            gap="test",
+            min_periods=1,
+        )
 
     error = "Cannot roll series. The specified window length, test, is not a valid offset alias."
     with pytest.raises(ValueError, match=error):
-        roll_series_with_gap(rolling_series_pd, window_size="test", gap="7D")
+        roll_series_with_gap(
+            rolling_series_pd,
+            window_size="test",
+            gap="7D",
+            min_periods=1,
+        )
```

Comment:
warning lots of black, pre-commit linting changes

Reply:
Do you know if there were any non linting changes to the actual test checks? I haven't seen any, but just want to make sure I'm not missing anything.

Reply:
Only change is specifying the min_periods parameter for `roll_series_with_gap`. I rolled the default parameters up to `apply_rolling_agg_to_series` so existing functions like `roll_series_with_gap` no longer need a default value.